### PR TITLE
Updated VPNSDKv2 to 2.2.44 version and done minor changes in build.ca…

### PR DESCRIPTION
**Pull request by renishnair-viper**
Updated VPNSDKv2 version to 2.2.44 version 
In build.cake updated SignToolSignSettings with CertSubjectName

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -15,6 +15,9 @@ var certificatePassword = Argument("CertPassword", "");
 var mygetApiKey = Argument("MyGetApiKey", "");
 var ApplicationName = Argument("ApplicationName","ConsumerVPN");
 var UsesEVCert = Argument<bool>("EVCert", false);
+var certSubject = ""; //Add your own cert subject
```
Comment (vimalraj-adikesavan) [inline build.cake]: Why is the subject empty?

**Review by rastapopoulos1 — APPROVED** [review]

File: build.cake [hunk 2]
```diff
@@ -27,10 +30,10 @@ var solutionDir = MakeAbsolute(Directory("."));
...
 {
-    Password = certificatePassword,
+    TimeStampUri = new Uri(TimeServer),
     DigestAlgorithm = SignToolDigestAlgorithm.Sha256,
     TimeStampDigestAlgorithm = SignToolDigestAlgorithm.Sha256,
-	ArgumentCustomization = args => args.Append("/tr", "http://timestamp.digicert.com")
+    CertSubjectName = certSubject	
 } : new SignToolSignSettings 
 {
     TimeStampUri = new Uri("http://timestamp.digicert.com"),
```
Comment (vimalraj-adikesavan) [inline build.cake]: Url can be replaced with TimeServer.

File: build.cake [hunk 3]
```diff
@@ -15,6 +15,9 @@ var certificatePassword = Argument("CertPassword", "");
 var mygetApiKey = Argument("MyGetApiKey", "");
 var ApplicationName = Argument("ApplicationName","ConsumerVPN");
 var UsesEVCert = Argument<bool>("EVCert", false);
+var certSubject = ""; //Add your own cert subject
+var TimeServer = "http://timestamp.sectigo.com";
```
Comment (vimalraj-adikesavan) [inline build.cake]: Suggest to name the variable as "TimeServerUrl"

File: build.cake (hunk 1 above)
Comment (renishnair-viper) [reply build.cake]: Now passing certSubject  as Argument.

File: build.cake (hunk 3 above)
Comment (renishnair-viper) [reply build.cake]: Updated to TimeServerUrl

File: build.cake (hunk 2 above)
Comment (renishnair-viper) [reply build.cake]: Updated with TimeServerUrl

**Review by rastapopoulos1 — APPROVED** [review]

**Review by vimalraj-adikesavan — APPROVED** [review]

**Pull request merged**